### PR TITLE
add the ability to force mentions to be rendered on top of the input

### PR DESCRIPTION
**Pull request by Bilb**
Currently, mentions can be rendered on top of the input/textarea only if the space below is not enough.
For my usecase, I wanted the mentions to be rendered on top of it always. So I added a boolean to do this.

### CODE REVIEW

**Comment by Bilb** [comment]
Any updates on this @jfschwarz

**Comment by Bilb** [comment]
Any updates on this @jfschwarz ? What can I do to get this merged?

**Review by jfschwarz — APPROVED** [review]
Sorry, for taking so long to review this. And thank you for bringing the PR to my attention again! 🙇

**Comment by Bilb** [comment]
@jfschwarz thanks for accepting the PR. I made the PR to update the types too and I think it needs a package maintainer approval for it to be merged: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53353

**Pull request merged**
